scss/civicrm/search-builder/_search-builder.scss

```diff
@@ -0,0 +1,44 @@
+.page-civicrm-contact-search-builder .crm-accordion-body {
+  #map-field td {
+    padding: 10px 20px 4px !important;
+  }
+
+  h3 {
+    background: $crm-accordion-header-color;
+    border-bottom: 1px solid $crm-dark-grayblue;
+    border-top: 1px solid $crm-dark-grayblue;
+    color: $gray-darker;
+    font-size: 13px;
+    margin-bottom: 0;
+    padding: 15px 20px;
+  }
+
+  .crm-search-builder-add-row .form-item,
+  .crm-search-builder-add-block {
+    text-decoration: none;
+
+    input.submit-link {
+      color: $btn-primary-bg !important;
```

Comment:
should be $brand-primary instead $btn-primary-bg this can be variation in future.
Why important! can we work without that and by just `.submit-link` instead of `input.submit-link`?